README.md

```diff
@@ -247,3 +254,20 @@ first part of the URL path, e.g.:
   "/otherdomain.biz": "http://10.0.1.4:5555",
 }
 ```
+
+## Using Redis
+
+If you require multiple instances of CHP to be running and kept in sync, you can use [redis] to to store the route table
```

Comment:
If your deployment runs multiple CHP instances, you may need to keep these instances in sync. You may use a storage provider, i.e. [redis],  to store the route table instead of the default which is storing the table in memory.